### ISSUE TITLE
Håndtering av bestillinger fra mange byer

### ISSUE THREAD

**Issue by aslakhol** [s0lvang/Karantenehjelpen#65]
**Hva ønsker du deg**
Når vi lanserer kan det være vi får bestillinger fra mange byer. Sortering basert på avstand som i #48 vil hjelpe på dette, men vi må ha noe inne før vi lanserer. 

Et forslag er et enkelt filter på by i listen.

**Comment by aslakhol** [comment]
Siden vi fikk inn #89 er dette ikke lenger stor pri. Men hadde vært nice med litt filtrering.

**Comment by Chr1stian** [comment]
Er ikke sikkert alle ønsker å dele posisjon, og det blir mer ryddig/oversiktlig med litt filter. Anbefales!

**Comment by awildfivreld** [comment]
Jeg trykket i utangspunktet nei på deling av posisjon (refleks), viste ikke at dette var en greie før jeg leste på Tiltaksgruppe GÅT. Burde kanskje også stå på nettsiden på en eller annen måte?

**Comment by FredrikAugust** [comment]
Tenker du at det bør stå hva posisjonen brukes til @awildfivreld?

**Comment by awildfivreld** [comment]
Ja, spesielt ettersom det ikke er noen annen form for filtrering for øyeblikket. Jeg ser for meg at det kan bli vanskelig å finne oppdrag hvis man da ikke har godkjent deling av posisjon.

**Comment by FredrikAugust** [comment]
Det er en implementasjon med et kart over alle ordre til review nå @awildfivreld #127.
